### DISCUSSION TITLE
How to claim previous 1.75 rewards?

### DISCUSSION THREAD

**Discussion by hhenrry501** [Chia-Network/chia-blockchain#17296] [Support]
I was self pooling and using 2.0.0 version on Windows 10. I won a block on Dec 10th 2023, and I didnt claim that 1.75 from "pooling" tab. Last week I upgraded to 2.1.3 version with v2 database, then I cant find that unclaimed 1.75 rewards, under the pooling tab, there was nothing but the option to join new pool, so I create a new self pool. But clearly it wasnt the way to claim my previous rewards. 
Please guide me on how to get that rewards back.
![WX20240111-35149 PM@2x](https://github.com/Chia-Network/chia-blockchain/assets/83000063/279ccb45-8331-4689-8844-54c8e41569af)

![WX20240111-51040 PM@2x](https://github.com/Chia-Network/chia-blockchain/assets/83000063/6df75e28-28d4-4b52-8e81-44fc8595844c)

**Comment by heget** [comment]
Try find the old config file, where you have information about you pool.
example:
pool:
--all this rows must be copied to new config--
prefer_ipv6: false

**Comment by BrandtH22** [comment]
Hey @hhenrry501 , what I would do is resync the wallet with the GUI settings / advanced / resync. This will close chia and upon reopening it will resync the wallet.

The resync will readd the pool information to the config and also identify the most recent coin states for any unclaimed rewards 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
